omero-ssh-systemd/Dockerfile

```diff
@@ -14,10 +14,19 @@ RUN sed -i \
 	's@session\s*required\s*pam_loginuid.so@session optional pam_loginuid.so@g' \
 	/etc/pam.d/sshd
 
+# To avoid error: sudo: sorry, you must have a tty to run sudo
+RUN sed -i -e "s/Defaults    requiretty.*/ #Defaults    requiretty/g" /etc/sudoers
+
 RUN useradd omero && \
 	echo 'omero:omero' | chpasswd && \
 	echo "omero ALL= (ALL) NOPASSWD: ALL" >> /etc/sudoers.d/omero
 
+RUN rm -f /lib/systemd/system/systemd*udev* ; \
+	rm -f /lib/systemd/system/getty.target;
+
+RUN yum install -y initscripts \
```

Comment:
to run `service` rather then `systemctl`, it makes all the devspace jobs and external scripts os independent, no?